internal/server/data/migrationdata/202207120000-postgres.sql

```diff
@@ -0,0 +1,887 @@
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE testing.identities_id_seq OWNER TO postgres;
+
+--
+-- Name: identities_id_seq; Type: SEQUENCE OWNED BY; Schema: testing; Owner: postgres
+--
+
+ALTER SEQUENCE testing.identities_id_seq OWNED BY testing.identities.id;
+
+
+--
+-- Name: migrations; Type: TABLE; Schema: testing; Owner: postgres
+--
+
+CREATE TABLE testing.migrations (
+    id character varying(255) NOT NULL
+);
+
+
+ALTER TABLE testing.migrations OWNER TO postgres;
+
+--
+-- Name: provider_users; Type: TABLE; Schema: testing; Owner: postgres
+--
+
+CREATE TABLE testing.provider_users (
+    identity_id bigint NOT NULL,
+    provider_id bigint NOT NULL,
+    id bigint NOT NULL,
+    created_at timestamp with time zone,
+    updated_at timestamp with time zone,
+    deleted_at timestamp with time zone,
+    email text,
+    groups text,
+    last_update timestamp with time zone,
+    redirect_url text,
+    access_token text,
+    refresh_token text,
+    expires_at timestamp with time zone
+);
+
+
+ALTER TABLE testing.provider_users OWNER TO postgres;
+
+--
+-- Name: provider_users_id_seq; Type: SEQUENCE; Schema: testing; Owner: postgres
+--
+
+CREATE SEQUENCE testing.provider_users_id_seq
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE testing.provider_users_id_seq OWNER TO postgres;
+
+--
+-- Name: provider_users_id_seq; Type: SEQUENCE OWNED BY; Schema: testing; Owner: postgres
+--
+
+ALTER SEQUENCE testing.provider_users_id_seq OWNED BY testing.provider_users.id;
+
+
+--
+-- Name: providers; Type: TABLE; Schema: testing; Owner: postgres
+--
+
+CREATE TABLE testing.providers (
+    id bigint NOT NULL,
+    created_at timestamp with time zone,
+    updated_at timestamp with time zone,
+    deleted_at timestamp with time zone,
+    name text,
+    kind text,
+    url text,
+    client_id text,
+    client_secret text,
+    auth_url text,
+    scopes text,
+    created_by bigint
+);
+
+
+ALTER TABLE testing.providers OWNER TO postgres;
+
+--
+-- Name: providers_id_seq; Type: SEQUENCE; Schema: testing; Owner: postgres
+--
+
+CREATE SEQUENCE testing.providers_id_seq
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE testing.providers_id_seq OWNER TO postgres;
+
+--
+-- Name: providers_id_seq; Type: SEQUENCE OWNED BY; Schema: testing; Owner: postgres
+--
+
+ALTER SEQUENCE testing.providers_id_seq OWNED BY testing.providers.id;
+
+
+--
+-- Name: settings; Type: TABLE; Schema: testing; Owner: postgres
+--
+
+CREATE TABLE testing.settings (
+    id bigint NOT NULL,
+    created_at timestamp with time zone,
+    updated_at timestamp with time zone,
+    deleted_at timestamp with time zone,
+    private_jwk bytea,
+    testing_jwk bytea,
+    lowercase_min bigint DEFAULT 0,
+    uppercase_min bigint DEFAULT 0,
+    number_min bigint DEFAULT 0,
+    symbol_min bigint DEFAULT 1,
+    length_min bigint DEFAULT 8
```

Comment:
I think you want these columns to not be in the table in order to test the migration